libs/application/templates/signature-collection/parliamentary-list-creation/src/lib/dataSchema.ts

```diff
@@ -0,0 +1,59 @@
+import { z } from 'zod'
+import { m } from './messages'
+import * as nationalId from 'kennitala'
+
+const nationalIdAndName = z.object({
+  name: z.string(),
+  nationalId: z.string().refine((v) => nationalId.isPerson(v), {
+    params: m.nationalIdValidationError,
+  }),
+})
+
+export const dataSchema = z.object({
+  /* Gagnaöflun */
+  approveExternalData: z.boolean().refine((v) => v),
+
+  /* Upplýsingar */
+  applicant: z.object({
+    name: z.string(),
+    nationalId: z.string(),
+    phone: z.string().min(11),
```

Comment:
Min 11, because of this, right?
```
1  2  3  4  5  6  7  8  9  10  11
+  3  5  4  8  2  2  1  4  2   9
```

Is this just supplementary to any validation present in the phoneNumberField to begin with?

Reply:
What about length 13 → "+354 822-1429"?

Reply:
So I'm wondering, if we're using the phoneNumberField to do validation, should we get out of its way with something like `z.string().min(1)` just to check that it is not empty?

Reply:
phoneNumberField doesnt do the validation